cli/src/template_builder.rs

```diff
@@ -326,6 +364,12 @@ impl<'a, L: TemplateLanguage<'a>> CoreTemplateBuildFnTable<'a, L> {
     }
 }
 
+impl<'a, L: TemplateLanguage<'a>> Default for CoreTemplateBuildFnTable<'a, L> {
+    fn default() -> Self {
+        Self::new()
+    }
```

Comment:
~`#[derive(Default)]` instead?~

Inline `::new()` in `::default()` if it's the main use case? If we don't need `::default()`, I would rename `::new()` to `::empty()` (so the caller will think a bit more whether he want `::builtin()` or `::empty()`.)

Reply:
Went with `empty()`